src/appforms/tests/tests/backbone/040-view02field05field_checkbox.js

```diff
@@ -0,0 +1,97 @@
+var assert = chai.assert;
+
+
+describe("Backbone - Checkbox Field View", function() {
+
+  beforeEach(function(done) {
+    var self = this;
+    var Form = appForm.models.Form;
+    new Form({
+      formId: "527d4539639f521e0a000004",
+      fromRemote: true
+    }, function(err, form) {
+      assert.ok(!err, "Expected No Error");
+      assert.ok(form, "Expected a form  model");
+
+      self.form = form;
+
+      self.formView = new FormView({
+        parentEl: $("<div></div>")
+      });
+
+      self.formView.loadForm({
+        form: form
+      }, function(err) {
+        assert.ok(!err, "Expected no error");
+        self.formView.render();
+
+
+        done();
+      });
+    });
+  });
+
+  it("Render Checkboxes field, blank option", function() {
```

Comment:
The style for mocha is usually like `it("should <test case>")` so the test output will read like "it should <test case>".  Maybe we should reword for readability?